src/components/crownButton.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+  <button
+    @click="$emit('click')"
```

Comment:
According to the docs at https://vuejs.org/v2/api/#inheritAttrs, it seems like you can remove the entire script block and all props, and all appropriate attributes will be applied to this root `button` element as expected (you don't need to set `inheritAttrs` to `false`).

I'm not sure if that will happen with event listeners, so in that case, you can put a `v-on="$listeners"` on the button.

Reply:
Thanks, that's what I was looking for.